lib/Jsrt/Core/JsrtContextCore.cpp

```diff
@@ -120,6 +120,36 @@ HRESULT ChakraCoreHostScriptContext::FetchImportedModule(Js::ModuleRecordBase* r
     return E_INVALIDARG;
 }
 
+HRESULT ChakraCoreHostScriptContext::FetchImportedModuleFromScript(JsSourceContext dwReferencingSourceContext, LPCOLESTR specifier, Js::ModuleRecordBase** dependentModuleRecord)
+{
+    if (fetchImportedModuleFromScriptCallback == nullptr)
+    {
+        return E_INVALIDARG;
+    }
+
+    HRESULT hr = NOERROR;
+    Js::JavascriptString* specifierVar = nullptr;
+    specifierVar = Js::JavascriptString::NewCopySz(specifier, GetScriptContext());
+    
+    if (FAILED(hr))
```

Comment:
NA in latest iteration (function refactored).

---
In reply to: [115328224](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r115328224) [](ancestors = 115328224)